tests/postgres/tfkpragma.nim

```diff
@@ -70,6 +72,6 @@ suite "``fk`` pragma":
     for inpCustomer in inpCustomers.mitems:
       dbConn.insert inpCustomer
 
-    dbConn.select(outCustomers, """"userid" = $1""", userA.id)
+    dbConn.select(outCustomers, """"userId" = $1""", userA.id)
```

Comment:
An example for sudden case-sensitivity, though that is when comparing column names and not during actual queries, not sure how much that matters.